docs/guide/src/pd/join-testnet/validator.md

```diff
@@ -0,0 +1,170 @@
+
+```toml
+mode = "validator"
+```
+
+Then you'll need to download our genesis JSON file and replace the automatically
+generated one.  You can find the genesis JSON files for all of our testnets [in
+our repository](https://github.com/penumbra-zone/penumbra/tree/main/testnets).
+
+The testnet directories are numbered in increasing order, find and download the
+latest JSON file and copy it to `\$HOME/.tendermint/config/genesis.json`.
+
+You can also download the genesis JSON directly from the testnet server, though it is nested
+so you need to grab a specific key:
+
+```console
+curl -X GET "http://testnet.penumbra.zone:26657/genesis" -H "accept: application/json" | jq '.result.genesis' > \$HOME/.tendermint/config/genesis.json
+```
+
+## Starting your node
+
+After configuration is complete, you're ready to start your node.
+
+First, start the `pd` binary:
+
+```console
+export RUST_LOG="warn,pd=debug,penumbra=debug,jmt=info" && \ # or some other logging level
+cargo run --release --bin pd start --rocks-path \$HOME/.rocksdb 
+```
+
+Then (perhaps in another terminal) start Tendermint:
+
+```console
+tendermint start
+```
+
+You should now be participating in the network as a fullnode. However your validator won't be visible
+to the chain yet, as the definition hasn't been uploaded.
+
+## Uploading your validator definition
+
+A validator definition contains fields defining metadata regarding your validator as well as initial funding streams.
+
+Create a `validator.json` file like so:
+
+```console
+\$ cargo run --release --bin pcli -- validator template-definition --file validator.json
+\$ cat validator.json
+{
+  "identity_key": "penumbravalid1g2huds8klwypzczfgx67j7zp6ntq2m5fxmctkf7ja96zn49d6s9qz72hu3",
+  "consensus_key": "Fodjg0m1kF/6uzcAZpRcLJswGf3EeNShLP2A+UCz8lw=",
+  "name": "",
+  "website": "",
+  "description": "",
+  "funding_streams": [
+    {
+      "address": "penumbrav1t1mw8270qtpgjy628fg97p2px45e860jtlw0nl3w5y7vq67qx697py9t8ppp3mhwfxv8kegg8wuny64nf60z966krx85cqznjpshqtngffpwnywtzqjklkg3qh7anxk368ywac9l",
+      "rate_bps": 100
+    }
+  ],
+  "sequence_number": 0
+}
+```
+
+and adjust the configurations as desired.
+
+By default `template-definition` will use a random consensus key that you won't have access to. Make sure to **set the `consensus_key` correctly** otherwise your instance of `tendermint` won't be using the key expected in the validator definition. You can get the correct value for `consensus_key` from your `tendermint` configs:
+
+```console
+\$ grep -A3 pub_key ~/.tendermint/config/priv_validator_key.json
+  "pub_key": {
+    "type": "tendermint/PubKeyEd25519",
+    "value": "Fodjg0m1kF/6uzcAZpRcLJswGf3EeNShLP2A+UCz8lw="
+  },
+```
+
+The `identity_key` value placed by the `template-definition` command in your `validator.json` will correspond to the key used by your wallet configs for `pcli`, i.e. `pcli` will have access to any balance delegated to the validator.
```

Comment:
Flagging this because it's not actually true -- the identity key for a validator doesn't get to control delegations, since delegations are totally non-custodial.

Reply:
Good catch, what would be more accurate to say? "The validator's address will be determined by the keys configured for the pcli wallet"?

Reply:
I put this:

>  The root of a validator's identity is their *identity key*.  Currently, `pcli`
reuses the spend authorization key in whatever wallet is active as the
validator's identity key.  This key is used to sign validator definitions that
update the configuration for a validator.